frmts/mem/memdataset.cpp

```diff
@@ -2311,13 +2423,38 @@ MEMMDArray::CreateAttribute(const std::string &osName,
     return newAttr;
 }
 
+/************************************************************************/
+/*                         DeleteAttribute()                            */
+/************************************************************************/
+
+bool MEMMDArray::DeleteAttribute(const std::string &osName,
```

Comment:
I'm happy enough with this small code duplication. It might require to add a new class to avoid it, and make MEMGroup and MEMMDArray derive from it, which would be overkill IMHO